honeybadger-java/src/main/java/io/honeybadger/reporter/dto/Backtrace.java

```diff
@@ -0,0 +1,56 @@
+package io.honeybadger.reporter.dto;
+
+import com.fasterxml.jackson.annotation.JacksonInject;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.honeybadger.reporter.config.ConfigContext;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+
+/**
+ * Class representing an ordered collection of backtrace elements.
+ * @author <a href="https://github.com/dekobon">Elijah Zupancic</a>
+ * @since 1.0.9
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Have you verified in the crywolf application that the shaded jars work properly with the Jackson annotations?

Reply:
I would be sure to test again after fixing the shade relocation issues.